handlers/validator.go

```diff
@@ -1075,12 +1069,9 @@ func ValidatorProposedBlocks(w http.ResponseWriter, r *http.Request) {
 
 	orderColumn := q.Get("order[0][column]")
 	orderByMap := map[string]string{
-		"0": "epoch",
-		"2": "status",
-		"5": "attestationscount",
-		"6": "depositscount",
-		"8": "voluntaryexitscount",
-		"9": "graffiti",
+		"0": "slot",
```

Comment:
Column 0 is "epoch".
If you would currently click on the arrows to sort by epoch it would sort by slot instead which would coincidentally give the same result I think.

Reply:
We could move the "order arrows" from "Epoch" to "Slot" but we can also just leave it as it is since when your sort by slot you also sort by epoch.

Reply:
Leaving it as is :)